certbot/cli.py

```diff
@@ -944,19 +953,17 @@ def _paths_parser(helpful):
         help="Logs directory.")
     add("paths", "--server", default=flag_default("server"),
         help=config_help("server"))
-    # overwrites server, handled in HelpfulArgumentParser.parse_args()
-    add("testing", "--test-cert", "--staging", action='store_true', dest='staging',
-        help='Use the staging server to obtain test (invalid) certs; equivalent'
-             ' to --server ' + constants.STAGING_URI)
 
 
 def _plugins_parsing(helpful, plugins):
+    # It's nuts, but there are two "plugins" topics.  Somehow this works
```

Comment:
I'm indifferent. I really don't think there are many use cases for the `plugins` subcommand at all. I've used it for debugging during plugin development occasionally, I guess. I imagine someone might use it to see what plugins they have, but I've never done that...